### ISSUE TITLE
Febrile urinary tract infection

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6651]
**Preferred term label:**
Febrile urinary tract infection

**Synonyms**
FUTI

**Definition (free text, please give PubMed ID)**
A urinary tract infection with fever.
PMID: 32548080

**Parent term (use hpo.jax.org/app)**
Abnormality of the urinary system physiology

**Comment by pnrobinson** [comment]
I think this is best conceptualized as a disease. Please see Mondo -- https://github.com/monarch-initiative/mondo
